lib/apiservers/engine/backends/container.go

```diff
@@ -1768,8 +1768,20 @@ func (c *Container) validateContainerLogsConfig(vc *viccontainer.VicContainer, c
 		since = time.Unix(s, n)
 	}
 
+	// TODO(jzt): this should not require an extra call to the portlayer. We should
+	// update container.DataVersion when we hydrate the container cache at VCH startup
+	// see https://github.com/vmware/vic/issues/4194
 	if config.Timestamps {
-		return unsupported("timestamps")
+		// check container DataVersion to make sure it's supported
+		params := containers.NewGetContainerInfoParams()
+		params.SetID(vc.ContainerID)
+		info, err := PortLayerClient().Containers.GetContainerInfo(params)
+		if err != nil {
+			return 0, 0, err
+		}
+		if info.Payload.DataVersion == 0 {
```

Comment:
suggest we create `lib/apiserver/engine/backends/feature-support.go` or similar and add some constant mappings:
```go
// Feature to min/max data version. -1 means unlimited
const (
LOG_SINCE_MIN = 1
LOG_SINCE_MAX = -1
)
```

or using functions 
```go
func logSinceSupported(data interface{}) bool {
...
}
```

This will prevent us having the checks scattered over the code and would provide a single reference location for the conditionals that apply to support.